packages/pnpm-sync-lib/src/utilities.ts

```diff
@@ -37,3 +37,9 @@ async function getFilesInDirectoryHelper(directory: string, returnFileList: ISyn
     });
   }
 }
+
+export const isPnpmV8 = (pnpmVersion: string): boolean => pnpmVersion.startsWith('8.');
```

Comment:
@octogonz I extracted the logic for checking pnpm version into three utility functions.